<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_add_hallnum">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>632</y>
     <width>133</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>增加HallNum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_remain_hallnum">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>632</y>
     <width>133</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>维持原状HallNum</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_changelist">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>733</width>
     <height>613</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>733</width>
     <height>613</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>733</width>
     <height>613</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>文件路径</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改方式</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>624</y>
     <width>413</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>当前修改涉及到Hall大厅修改，请仔细确认是否需要提升大厅框架号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
